### ISSUE TITLE
How about an UWP version?

### ISSUE THREAD

**Issue by devinfluencer** [Microsoft/ConferenceVision#8]
Or am I the last one using Windows Mobile?

**Comment by davidortinau** [comment]
@devinfluencer I saw a few at Build this week. Deep is the love.

I did implement this as a phone first demo, and the UWP project is still in the repository. I'm open to adding it back in if someone can PR the bits we need.

- [ ] Camera
- [ ] Media folder path
- [ ] Twitter share

Do you have any of those?

**Comment by devinfluencer** [comment]
I think the Media folder path should be the easiest one of these. Camera should be relatively easy if the interface functions are described well enough which I didn't check yet. I don't know about twitter APIs.
I assume the missing bits will be added if you prepare the solution. I'd be glad to contribute if I can.

When I tried to add an UWP project myself I realized .net standard 2.0 is used so my Windows Mobile Lumia 950XL would be left behind. What reason disallow a downgrade of the .net standard version?
